pkg/backup/backup.go

```diff
@@ -284,7 +288,7 @@ func (kb *kubernetesBackupper) Backup(log logrus.FieldLogger, backupRequest *Req
 
 func (kb *kubernetesBackupper) writeBackupVersion(tw *tar.Writer) error {
 	versionFile := filepath.Join(api.MetadataDir, "version")
-	versionString := fmt.Sprintf("%d\n", BackupVersion)
+	versionString := fmt.Sprintf("%v\n", BackupFormatVersion)
```

Comment:
Minor, but I'd change this to `%s` instead of `%v` to be sure it's a string instead of any Go value.